File: Java Programs/evenOdd.java
/* JAVA PROGRAM TO CHECK WHETHER NUMBER IS EVEN OR ODD */
import java.util.*;
public class evenOdd
{
	public static void check(int n)
	{
		if(n%2==0)
		{
			System.out.println("Number "+n+" Is Even");
		}
		else
		{
			System.out.println("Number "+n+" Is Odd");
		}
	}
	public static void main(String[] args)
	{
		int n;
		Scanner sc=new Scanner(System.in);
		System.out.println("Enter Number=");
		n=sc.nextInt();
		check(n);
	}
}

File: Java Programs/evenOdd.cpp
/* CPP PROGRAM TO CHECK WHETHER NUMBER IS EVEN OR ODD */
#include<iostream>
using namespace std;
void check(int n)
{
	if(n%2==0)
	{
		cout<<"Number "<<n<<" Is Even";
	}
	else
	{
		cout<<"Number "<<n<<" Is Odd";
	}
}
int main()
{
	int n;
	cout<<"Enter Number=";
	cin>>n;
	check(n);
}